configuration/monitoring.md

```diff
@@ -0,0 +1,20 @@
+## Setting up monitoring using Grafana
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/alertmanager.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheus.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/podmonitor.crd.yaml
+​
+sleep 5 # wait for crds to be created
+​
+helm upgrade --install monitoring stable/prometheus-operator \
+  --version 6.9.3 \
+  --namespace monitoring \
+  --set prometheusOperator.createCustomResource=false \
+  --set grafana.service.type=LoadBalancer \
+  --set grafana.adminPassword="SOMETHING_TO_BE_CHANGED"
```

Comment:
For consistency with the ones presented above, maybe this adminPassword should say "PLEASE_CHANGE_ME"?
I am just thinking that, myself, as a developer, if I skim through the text and I see some CHANGE_ME in some credentials, I will try to do a find to understand what else needs to be replaced.